src/components/TimePicker/__test__/timeSelect.spec.js

```diff
@@ -79,33 +92,392 @@ describe('<TimeSelect/>', () => {
+        minutesInput.simulate('change', { target: { value: '20' } });
+        container.simulate('keyDown', { keyCode: 39 });
+        container.simulate('keyDown', { keyCode: 8 });
+        container.simulate('keyDown', { keyCode: 40 });
+        expect(
+            component
+                .find('input')
+                .at(1)
+                .prop('value'),
+        ).toBe('59');
+    });
+    it('should call event.stopPropagation when press enter key', () => {
+        const stopPropagationMockFn = jest.fn();
+        const component = mount(<TimeSelect />);
+        const container = component.find('div[role="presentation"]');
+        container.simulate('keyDown', {
+            keyCode: 13,
+            stopPropagation: stopPropagationMockFn,
+        });
+        expect(stopPropagationMockFn).toHaveBeenCalledTimes(1);
+    });
+    it('should call event.preventDefault when press enter key', () => {
+        const preventDefaultMockFn = jest.fn();
+        const component = mount(<TimeSelect />);
+        const container = component.find('div[role="presentation"]');
+        container.simulate('keyDown', {
+            keyCode: 13,
+            preventDefault: preventDefaultMockFn,
+        });
+        expect(preventDefaultMockFn).toHaveBeenCalledTimes(1);
+    });
+    it('should call onCloseModal when when press enter key', () => {
+        const onCloseModalMockFn = jest.fn();
+        const component = mount(<TimeSelect onCloseModal={onCloseModalMockFn} />);
+        const container = component.find('div[role="presentation"]');
+        container.simulate('keyDown', { keyCode: 13 });
+        expect(onCloseModalMockFn).toHaveBeenCalledTimes(1);
+    });
+    it('should call onChange with the right value when press enter key while hour, minutes and ampm has value', () => {
+        const onChangeMockFn = jest.fn();
+        const component = mount(<TimeSelect onChange={onChangeMockFn} />);
+        const container = component.find('div[role="presentation"]');
+        const hourInput = component.find('input').at(0);
+        const minutesInput = component.find('input').at(1);
+        hourInput.simulate('change', { target: { value: '12' } });
+        minutesInput.simulate('change', { target: { value: '14' } });
+        container.simulate('keyDown', { keyCode: 39 });
+        container.simulate('keyDown', { keyCode: 39 });
+        container.simulate('keyDown', { keyCode: 38 });
+        container.simulate('keyDown', { keyCode: 13 });
+        expect(onChangeMockFn).toHaveBeenCalledWith('12:14');
+    });
+    it('should not call onChange when press enter key while hour, minutes and ampm has not value', () => {
+        const onChangeMockFn = jest.fn();
+        const component = mount(<TimeSelect onChange={onChangeMockFn} />);
+        const container = component.find('div[role="presentation"]');
+        container.simulate('keyDown', { keyCode: 13 });
+        expect(onChangeMockFn).toHaveBeenCalledTimes(0);
     });
```

Comment:
more tests to add:
- should call onCloseModal when click the cancel button
and add the cases when press ok button